include/crocoddyl/multibody/actuations/multicopter-base.hpp

```diff
@@ -0,0 +1,84 @@
+///////////////////////////////////////////////////////////////////////////////
+// BSD 3-Clause License
+//
+// Copyright (C) 2018-2020, LAAS-CNRS, IRI: CSIC-UPC
+// Copyright note valid unless otherwise stated in individual files.
+// All rights reserved.
+///////////////////////////////////////////////////////////////////////////////
+
+#ifndef CROCODDYL_MULTIBODY_ACTUATIONS_MULTICOPTER_BASE_HPP_
+#define CROCODDYL_MULTIBODY_ACTUATIONS_MULTICOPTER_BASE_HPP_
+
+#include "crocoddyl/multibody/fwd.hpp"
+#include "crocoddyl/core/utils/exception.hpp"
+#include "crocoddyl/core/actuation-base.hpp"
+#include "crocoddyl/multibody/states/multibody.hpp"
+
+namespace crocoddyl {
+template <typename _Scalar>
+class ActuationModelMultiCopterBaseTpl : public ActuationModelAbstractTpl<_Scalar> {
+ public:
+  typedef _Scalar Scalar;
+  typedef MathBaseTpl<Scalar> MathBase;
+  typedef ActuationModelAbstractTpl<Scalar> Base;
+  typedef StateMultibodyTpl<Scalar> StateMultibody;
+  typedef ActuationDataAbstractTpl<Scalar> ActuationDataAbstract;
+  typedef typename MathBase::VectorXs VectorXs;
+  typedef typename MathBase::MatrixXs MatrixXs;
+
+  ActuationModelMultiCopterBaseTpl(boost::shared_ptr<StateMultibody> state, const std::size_t& n_rotors,
```

Comment:
It would be good to add some comments what kind of MultiCopter actuation model this is - e.g. a reference to a paper or at least inline code comments on how it works, what the assumptions and notation are, etc. 
[E.g. the Mellinger model is fairly common so if this were one of that kind, a reference would be sufficient with notes of modifications]

Reply:
I agree in explaining a little bit the model. I think that in the Mellinger's model the control inputs are considered to be the net force and moments applied to the body due to the force of the different propellers., i.e. `data->tau`.
Here, instead, we consider the force of each propeller.

Reply:
Thank you @PepMS, this is really helpful for anybody looking at the code at a later stage :-)

Reply:
@PepMS and @wxmerkt I didn't notice this thread. I just merged this branch.

@PepMS could you create another PR tackling this issue?

Reply:
Sure!